File: backend/src/Fix.Application/UseCases/Organizations/Dtos/OrganizationDto.cs
namespace Fix.Application.Organizations.Dtos;

/// <param name="IsInternal">Organização nativa da FIX (equipe interna).</param>
/// <param name="InternalAccess">O usuário vê a organização pelo acesso interno da equipe FIX, sem ser membro.</param>
public sealed record OrganizationDto(Guid Id, string Name, string Slug, bool IsInternal = false, bool InternalAccess = false);

File: frontend/bff/src/application/organizations/dtos/organization.dto.ts
export interface OrganizationDto {
  id: string;
  name: string;
  slug: string;
  /** Organização nativa da FIX (equipe interna). */
  isInternal: boolean;
  /** O usuário vê a organização pelo acesso interno FIX, sem ser membro (suporte: vê e edita, não decide). */
  internalAccess: boolean;
}
